src/ra/groupavg.h

```diff
@@ -0,0 +1,85 @@
+#ifndef RA_GroupAvg_H_
+#define RA_GroupAvg_H_
+
+#include <type_traits>
+#include <utility>
+
+#include <iostream>
+
+#include "range/v3/all.hpp"
+
+/* This operator should only be used following ra::groupby */
+
+namespace fluent {
+namespace ra {
+
+template <typename PhysicalChild, std::size_t Is>
```

Comment:
Naming a non-variadic template `Is` is confusing when we've been using elsewhere as a variadic template. Should be called `I`.